### PR TITLE
fix: code clean up for 4ShalfFeatured, 3SautoSlider, twentyStory4C(ne…

### PR DESCRIPTION
**Pull request by supreeth-spiger**
code clean up for following components:
> 4ShalfFeatured, 
> 3SautoSlider,
> twentyStory4C(new component)

### CODE REVIEW

File: src/components/atoms/BigStoryCardAutoSlider/bigStoryCardAutoSlider.m.css [hunk 1]
```diff
@@ -46,8 +49,9 @@
 }
 
 .content {
-  margin: 0px;
+  margin: var(--space2) 0;
   color: var(--white);
+  font-size: var(--space2);
```
Comment (nandap4790) [inline src/components/atoms/BigStoryCardAutoSlider/bigStoryCardAutoSlider.m.css]: i think for the fonts we should be having a separate file, space2 for fonts seems a bit out of context

File: src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css [hunk 2]
```diff
@@ -43,15 +43,13 @@
   /* @include font-styles(h2); */
   font-size: var(--large);
```
Comment (nandap4790) [inline src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css:40]: we need something like this

Comment (nandap4790) [reply src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css:40]: and we can remove the comments

File: src/components/rows/TwentyStory4c/index.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import PropTypes from "prop-types";
+import produce from "immer";
+import { getPagePath } from "../../utils/collection";
```
Comment (nandap4790) [inline src/components/rows/TwentyStory4c/index.js:5]: line break before 4

File: src/components/rows/TwentyStory4c/index.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+import CollectionLink from "../../utils/generate-collection-link";
+
```
Comment (nandap4790) [inline src/components/rows/TwentyStory4c/index.js]: not required here, line break

File: src/components/rows/TwentyStory4c/stories.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { withStore } from "../../../../storybook/index";
+import produce from "immer";
+import TwentyStory4C from "./index";
```
Comment (nandap4790) [inline src/components/rows/TwentyStory4c/stories.js]: line break and split

File: src/components/rows/TwentyStory4c/twentyStory4c.m.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  padding: var(--space1_5) 0 var(--space3);
+  @media (min-width: 768px) {
+    padding: var(--space3) 0;
+  }
+}
+.base :global .sub-collection-headline {
+  margin: 0px;
+  align-self: end;
+  color: var(--content-color);
+  font-size: var(--space3);
```
Comment (nandap4790) [inline src/components/rows/TwentyStory4c/twentyStory4c.m.css]: here as well

File: src/components/rows/TwentyStory4c/twentyStory4c.m.css [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  grid-column-gap: var(--space2);
+  grid-row-gap: var(--space1_5);
+  grid-template-columns: 1fr;
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto 1fr auto;
+    grid-auto-flow: column;
+    grid-row-gap: var(--space2);
+  }
+  align-items: flex-start;
```
Comment (nandap4790) [inline src/components/rows/TwentyStory4c/twentyStory4c.m.css]: move this before the media query

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: src/components/atoms/BigStoryCardAutoSlider/bigStoryCardAutoSlider.m.css (hunk 1 above)
Comment (supreeth-spiger) [reply src/components/atoms/BigStoryCardAutoSlider/bigStoryCardAutoSlider.m.css]: changes made

File: src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css (hunk 2 above)
Comment (supreeth-spiger) [reply src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css:40]: done

File: src/components/rows/TwentyStory4c/index.js (hunk 3 above)
Comment (supreeth-spiger) [reply src/components/rows/TwentyStory4c/index.js:5]: done

File: src/components/rows/TwentyStory4c/index.js (hunk 4 above)
Comment (supreeth-spiger) [reply src/components/rows/TwentyStory4c/index.js]: done

File: src/components/rows/TwentyStory4c/stories.js (hunk 5 above)
Comment (supreeth-spiger) [reply src/components/rows/TwentyStory4c/stories.js]: done

File: src/components/rows/TwentyStory4c/twentyStory4c.m.css (hunk 6 above)
Comment (supreeth-spiger) [reply src/components/rows/TwentyStory4c/twentyStory4c.m.css]: done

File: src/components/rows/TwentyStory4c/twentyStory4c.m.css (hunk 7 above)
Comment (supreeth-spiger) [reply src/components/rows/TwentyStory4c/twentyStory4c.m.css]: done

File: src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css [hunk 8]
```diff
@@ -40,18 +40,15 @@
...
-  /* @include font-styles(h2); */
   font-size: var(--large);
   color: var(--gray);
-  padding-left: var(--space2);
   padding-right: var(--space2);
   align-self: flex-start;
   display: flex;
   line-height: 1;
   @media (min-width: 768px) {
     /* @include font-styles(jumbo); */
```
Comment (nandap4790) [inline src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css]: commented fonts can be removed

**Review by nandap4790 — APPROVED** [review]

File: src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css (hunk 8 above)
Comment (supreeth-spiger) [reply src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css]: change made

**Pull request merged**